include/RAJA/policy/openmp_target/reduce.hpp

```diff
@@ -100,8 +101,8 @@ struct Reduce_Data
    *
    *  allocates data on the host and device and initializes values to default
    */
-  Reduce_Data(T /*defaultValue*/, T identityValue, Offload_Info &info)
-      : value(identityValue),
+  Reduce_Data(T initValue, T identityValue, Offload_Info &info)
+     : value(initValue), identity(identityValue),
```

Comment:
Previously value was initialized using identityValue, is my change here reasonable?

Reply:
This seems fine, and is conformant to the CUDA reset.